print_macros/src/lib.rs

```diff
@@ -0,0 +1,78 @@
+#![no_std]
+
+/*! Runtime debugging tools.
+
+This is a low-level crate intended to use from libtock.
+If you want to use the macros in a crate that is not part of libtock,
+import `libtock::print_macros` instead.
+
+When using within libtock, add `libtock_console` dependency.
```

Comment:
I think you should replace `libtock` with `libtock-rs` in this comment. `libtock` is a particular crate within the `libtock-rs` project, and it already depends on `libtock_console`.

Reply:
@dcz-self Are you still interested in working on this PR and your other `libtock-rs` PRs?